tests/specs/mcd/vow-cage-surplus-pass-rough-spec.k

```diff
@@ -73,31 +73,31 @@ module VOW-CAGE-SURPLUS-PASS-ROUGH-SPEC
               <code> Vow_bin_runtime </code>
               <storage> ACCT_ID_STORAGE => ACCT_ID_STORAGE [ #Vow.live <- 0 ] [ #Vow.Sin <- 0 ] [ #Vow.Ash <- 0 ] </storage>
               <origStorage> ACCT_ID_ORIG_STORAGE </origStorage>
-              <nonce> _Nonce_Vow => ?_ </nonce>
+              <nonce> Nonce_Vow => ?_ </nonce>
             </account>
             <account>
               <acctID> Vat </acctID>
               <balance> Vat_balance </balance>
               <code> Vat_bin_runtime </code>
               <storage> Vat_STORAGE => Vat_STORAGE [ #Vat.dai[Flapper] <- 0 ] [ #Vat.dai[ACCT_ID] <- (Dai_v +Int Dai_f) -Int Sin_v ] [ #Vat.sin[ACCT_ID] <- 0 ] [ #Vat.vice <- Vice -Int Sin_v ] [ #Vat.debt <- Debt -Int Sin_v ] </storage>
               <origStorage> Vat_ORIG_STORAGE </origStorage>
-              <nonce> _Nonce_Vat => ?_ </nonce>
+              <nonce> Nonce_Vat => ?_ </nonce>
             </account>
             <account>
               <acctID> Flapper </acctID>
               <balance> Flapper_balance </balance>
               <code> Flapper_bin_runtime </code>
               <storage> Flapper_STORAGE => Flapper_STORAGE [ #Flapper.live <- 0 ] </storage>
               <origStorage> Flapper_ORIG_STORAGE </origStorage>
-              <nonce> _Nonce_Flapper => ?_ </nonce>
+              <nonce> Nonce_Flapper => ?_ </nonce>
             </account>
             <account>
               <acctID> Flopper </acctID>
               <balance> Flopper_balance </balance>
               <code> Flopper_bin_runtime </code>
               <storage> Flopper_STORAGE => Flopper_STORAGE [ #Flopper.live <- 0 ] [ #Flopper.vow <- ACCT_ID ] </storage>
               <origStorage> Flopper_ORIG_STORAGE </origStorage>
-              <nonce> _Nonce_Flopper => ?_ </nonce>
+              <nonce> Nonce_Flopper => ?_ </nonce>
```

Comment:
Is there any reason that we're not using the more specific `<nonce> Nonce_Flopper </nonce>` instead of saying taht it could go to anything? The other nonces in this file do the same.

Reply:
For example, does `Nonce_Flopper => Nonce_Flopper +Int 1` work?